migrations/202202151654-create-user.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("UserType", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      Type: {
+        defaultValue: "Customer",
+        type: Sequelize.ENUM("Staff(admin & kitchen)", "Customer", "Driver"),
```

Comment:
Good use of enum. 
Naming could be better. 
It's better to not use any special characters in the naming as well.

Reply:
OK, I shall remove the ampersand